Source/JavaScriptCore/jsc.cpp

```diff
@@ -2084,7 +2084,7 @@ JSC_DEFINE_HOST_FUNCTION(functionWriteFile, (JSGlobalObject* globalObject, CallF
 
     int size = std::visit(WTF::makeVisitor([&](const String& string) {
         CString utf8 = string.utf8();
-        return FileSystem::writeToFile(handle, utf8.span());
+        return FileSystem::writeToFile(handle, byteCast<uint8_t>(utf8.span()));
```

Comment:
What would you think about adding a `span8()` to `CString` which returns a `LChar` span? We have this pattern in quite a few places already.

Reply:
It's tempting but I'm challenged by the ambiguity: Does it mean "convert to Latin1" or "give me the Latin1 you already have?"

CString can't quite do either of these operations because it doesn't offer conversion and it doesn't know what encoding it already has.

For `String` and related types, at least they `ASSERT` if you ask for `span8()` and they don't have it.

I think `span8()` would be great for `CString` if we could `ASSERT` correctness (maybe by storing a debug-only encoding field, set by the constructor?), or ensure it by template parameter.

Reply:
...in this case, we have UTF-8, so I guess `span8()` would need to `ASSERT` or fail to compile.

But if we had a verified `span8()`, maybe we could change `span()` to return `uint8_t`, and use it here.

Reply:
...and maybe we could also add `span16()` and `spanUTF8()`.